ui/components/data-sources/options/PostgresOptions.js

```diff
@@ -63,7 +63,7 @@ const PostgresOptions = ({ options, onOptionsChange, isDisabled, validations })
                 />
             </Col>
         </FormGroup>
-        <FormGroup controlId="password" validationState={validations.password}>
+        <FormGroup controlId="password" validationState="success">
```

Comment:
Don't restrain extensibility, we don't know what's the future validation for password. Let's better delegate this value to onPasswordChange or in any case, the initial state.